applications/nrf5340_audio/src/bluetooth/Kconfig.defaults

```diff
@@ -0,0 +1,164 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+config BT_AUDIO
+	bool
+	default y
+
+# Not supported on Packetcraft controller
+config BT_ATT_ENFORCE_FLOW
+	bool
+	default n
+
+# Disable Zephyr HCI Vendor-Specific extensions
+config BT_HCI_VS_EXT
+	bool
+	default n
+
+# Prevent the MAC address from changing after every reboot
+config BT_PRIVACY
+	bool
+	default n
+
+config BT_EXT_ADV
+	bool
+	default y
+
+config BT_ISO_RX_MTU
+	int
+	default 190
+
+config BT_ISO_TX_MTU
+	int
+	default 190
+
+if TRANSPORT_CIS # Depends on TRANSPORT_CIS
+
+config BT_DEVICE_NAME
+	string
+	default "NRF5340_AUDIO"
+
+config BT_GATT_CLIENT
+	bool
+	default y
+
+config BT_GATT_DYNAMIC_DB
+	bool
+	default y
+
+config BT_BONDABLE
+	bool
+	default n
+
+config BT_SMP
+	bool
+	default y
+
+config BT_GAP_AUTO_UPDATE_CONN_PARAMS
+	bool
+	default n
+
+config BT_AUTO_PHY_UPDATE
+	bool
+	default n
+
+config BT_AUTO_DATA_LEN_UPDATE
+	bool
+	default n
+
+config BT_L2CAP_TX_BUF_COUNT
+	int
+	default 6
+
+config BT_BUF_ACL_RX_SIZE
+	int
+	default 259
+
+# HEADSET
+if AUDIO_DEV = 1
+
+config BT_AUDIO_UNICAST_SERVER
+	bool
+	default y
+
+config BT_MAX_CONN
+	int
+	default 1
+
+config BT_ISO_MAX_CHAN
+	int
+	default 1
+
+config BT_ASCS_ASE_SNK_COUNT
+	int
+	default 1
+
+config BT_ASCS_ASE_SRC_COUNT
+	int
+	default 0
+
+config BT_PERIPHERAL
+	bool
+	default y
+
+config BT_GAP_PERIPHERAL_PREF_PARAMS
+	bool
+	default n
+
+endif # AUDIO_DEV = 1 (HEADSET)
+
+# GATEWAY
+if AUDIO_DEV = 2
+
+config BT_AUDIO_UNICAST_CLIENT
+	bool
+	default y
+
+config BT_MAX_CONN
+	int
+	default 1
+
+config BT_ISO_MAX_CHAN
+	int
+	default 1
```

Comment:
This already defaults to 1

Reply:
Keeping this since we are going to have dual link soon